pkg/xds/experimental/client/gcp/doc.go

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+// Package gcp provides helpers and cell definition specific to Google Cloud Project.
+package gcp
```

Comment:
Where is this package being used? Maybe simpler to just leave it out of scope unless we need it for this xDS client

Reply:
It is not used, but I want to keep it alongside the xDS client implementation in order to give an example of real-life usage, so others could easily start with their own platform-specific implementations